src/components/ui/Pool/PoolDeposit.tsx

```
@@ -413,19 +413,19 @@ const PoolDeposit = ({
+                  disabled={BigInt(ethers.parseUnits(String(inputValueToken1 || 0), selectedPool.token1.decimals)) > balanceToken1}
                   className="w-full mt-4"
                   onClick={
                     isConnected
                       ? () => {
-                          onDeposit(inputValueToken0, inputValueToken1);
-                          setSliderValue(50);
-                        }
+                        onDeposit(inputValueToken0, inputValueToken1);
+                        setSliderValue(50);
+                      }
                       : () => openConnectModal && openConnectModal()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)